singer-io/tap-pepperjam
# v.0.0.4 Fix api_key, date window issues

# Description of change
Fix issue where `api_key` is appearing in logs. Fix issue where `transaction_history` date window cannot exceed 28 days.

# Manual QA steps
Adjusted `date_window_days` param to 40. Ran discovery, singer-check-tap, initial load, and incremental sync for all endpoints (incl. transaction_history). Verified `transaction_history` loaded w/ 28 day windows. Also, verified `api_key` value did not appear in logs.
 
# Risks
Low - currently in Alpha testing with Pepperjam.
 
# Rollback steps
Revert to v.0.0.3
